models/nft/ethereum/nft_ethereum_aggregators_markers.sql

```diff
@@ -9,29 +10,30 @@
 }}
 
  WITH reservoir AS (
-    SELECT distinct substring(unhex(regexp_replace(data, '^.*00', '')), 2, length(unhex(regexp_replace(data, '^.*00', '')))-2) AS router_website
-    , regexp_replace(data, '^.*00', '') AS hash_marker
+    SELECT distinct from_utf8(from_hex(substring(regexp_replace(cast(data as varchar), '^.*00', ''), 3, length(regexp_replace(cast(data as varchar), '^.*00', ''))-4))) AS router_website
+    , from_hex(regexp_replace(cast(data as varchar), '^.*00', '')) AS hash_marker
     FROM {{ source('ethereum','transactions') }}
     WHERE to IN (
-        '0x00000000006c3852cbef3e08e8df289169ede581', --seaport
-        '0x74312363e45dcaba76c59ec49a7aa8a65a67eed3', --x2y2
-        '0x59728544b08ab483533076417fbbb2fd0b17ce3a', --looksrare
-        '0x9ebfb53fa8526906738856848a27cb11b0285c3f'  --reservoir
+        0x00000000006c3852cbef3e08e8df289169ede581, --seaport
+        0x74312363e45dcaba76c59ec49a7aa8a65a67eed3, --x2y2
+        0x59728544b08ab483533076417fbbb2fd0b17ce3a, --looksrare
+        0x9ebfb53fa8526906738856848a27cb11b0285c3f  --reservoir
     )
-    AND `RIGHT`(data, 2) = '1f'
-    AND `LEFT`(regexp_replace(data, '^.*00', ''), 2)='1f'
-    AND regexp_replace(data, '^.*00', '') != '1f'
-    AND length(regexp_replace(data, '^.*00', ''))%2 = 0
-    AND block_time > '2022-10-15'
+    AND substr(cast(data as varchar),length(cast(data as varchar))-1, 2) = '1f'
+    AND substr(regexp_replace(cast(data as varchar), '^.*00', ''),1, 2)='1f'
+    AND regexp_replace(cast(data as varchar), '^.*00', '') != '1f'
```

Comment:
This can be done with `bytearray_ltrim`. `select bytearray_ltrim(0x00001f) = 0x1f` evaluates to true.